src/DetailsView/actions/tab-stop-requirement-action-message-creator.ts

```diff
@@ -123,4 +124,15 @@ export class TabStopRequirementActionMessageCreator extends DevToolActionMessage
             payload,
         });
     };
+
+    public updateNeedToCollectTabbingResults = (needToCollectTabbingResults: boolean) => {
+        const payload: UpdateNeedToCollectTabbingResultsPayload = {
```

Comment:
This function is missing accompanying unit tests - jest code cov shows uncovered lines "129-133"